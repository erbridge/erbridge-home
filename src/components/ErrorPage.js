import React from 'react';

const ErrorPage = () => (
  <div>
    <h1>Oops!</h1>
    <p>Looks like something went wrong. Check the URL and try again!</p>
  </div>
);

export default ErrorPage;
