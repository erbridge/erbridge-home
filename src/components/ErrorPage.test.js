import { shallow } from 'enzyme';
import React from 'react';

import ErrorPage from './ErrorPage';

it('renders correctly', () => {
  expect(shallow(<ErrorPage />)).toMatchSnapshot();
});
