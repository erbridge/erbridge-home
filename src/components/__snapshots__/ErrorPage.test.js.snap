// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Oops!
  </h1>
  <p>
    Looks like something went wrong. Check the URL and try again!
  </p>
</div>
`;
